src/components/Articles.jsx

```diff
@@ -1,28 +1,33 @@
-import { useState, useEffect} from "react"
-import getArticles from "../../api-get"
+import { useState } from "react"
+import {useEffect} from "react"
+import {getArticles} from "../../api-get"
+import '../../src/Styling/article.css'
+
 
 export const Articles = () => {
     
-    const [articleList, setArticleList] = useState([])
-
-    useEffect (() => {
-        getArticles()
-        .then((body) =>{
-            console.log(body)
-            setArticleList(body)
-            console.log(body)
-        })
-        .catch((err) =>{
-            console.error(err)
-        })
-    }, [])
+const [articleList, setArticleList] = useState([])
 
-    return (
-        <div>
-            {articleList.map((article) =>{
-                <p>{article}</p>
-                console.log('article', article)
-            })}
-        </div>
-    )
+useEffect (() => {
+    getArticles()
+    .then((body) =>{
+        setArticleList(body.articles)
+    }).catch((err) =>{
+        console.error(err)
+    })
+}, [])
+return (
+    <div className="articles-container">
+        {articleList.map((article) => (
+            <div className="article-card" key={article.article_id}>
```

Comment:
consider making the whole div or at least the image of the article clickable - users will most likely want to be able to click anywhere on the article card to get to the article rather than having to specifically go to the read more link at the end as this is easier and quicker